playbooks/installation/index.adoc

```diff
@@ -592,7 +592,35 @@ TODO
 
 ==== Syncing Images using Satellite 6
 
-TODO
+To sync Docker images in Satellite all you need to do is create a product and in that product create an repository for each image.
+
+----
+hammer product create --name "OCP Docker Images" --organization "Lab"
+hammer repository create --name "openshift3/ose" --organization-id "Lab" --content-type docker --url "https://registry.access.redhat.com" --docker-upstream-name "openshift3/ose" --product "OCP Docker Images"
+----
+
+NOTE: Satellite will sync ALL tags for each image repository you create so it might be space intensive.
+
+There is a script available at https://github.com/redhat-cop/openshift-toolkit/tree/master/satellite/populate-docker.sh that can be used to save time.
+
+Update ORG_ID and PRODUCT_NAME if desired in the above script and run it. Note that you will need to have configured hammer authentication as described in the https://access.redhat.com/documentation/en-us/red_hat_satellite/6.2/html-single/hammer_cli_guide/#sect-CLI_Guide-Authentication[documentation].
+
+----
+$ ./populate-docker.sh
+----
+
+By adding the below 2 lines in your inventory file you configure OpenShift to consume the images from Satellite that was synced above.
+
+----
+oreg_url=satellite.example.com:5000/lab-ocp_docker_images-openshift3_ose-${component}:${version}
+cli_docker_additional_registries=satellite.example.com:5000
```

Comment:
Is cli_docker_additional_registries variable still valid? I dont see it in anywhere in the openshift_ansible repo...

Reply:
Ah, no should be openshift_ now, I fix.